packages/tkg-clusterclass-vsphere/bundle/config/upstream/base.yaml

```diff
@@ -1502,6 +1502,12 @@ spec:
       - op: add
         path: /spec/template/spec/joinConfiguration/nodeRegistration/kubeletExtraArgs/register-with-taints
         value: os=windows:NoSchedule
+      - op: add
+        path: /spec/template/spec/files/-
+        value:
+          path: 'C:\k\prevent_windows_updates.ps1'
+          content: |
+            Set-Service -Name "wuauserv" -StartupType Disabled -Status Stopped
       - op: add
         path: /spec/template/spec/postKubeadmCommands/-
         value: powershell c:/k/prevent_windows_updates.ps1 -ExecutionPolicy Bypass
```

Comment:
Is powershell ok with the forward slashes here? The file just added uses backslashes.

Reply:
yes, that's fine.